src/components/Banner/AnonBannerSticky.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { DesktopOnly, MobileOnly } from './../Responsive'
+import BannerDesktop from './BannerDesktop'
+import BannerMobile from './BannerMobile'
+import StickyBannerContent from './StickyBannerContent'
+
+class AnonBannerSticky extends React.PureComponent {
+  state = { isVisible: false }
+
+  componentDidMount () {
+    this.viewportHeight = window.innerHeight
+  }
+
+  componentDidUpdate (prevProps) {
+    if (!prevProps.bannerStaticRef && this.props.bannerStaticRef) {
+      this.checkVisibility()
+      this.timer = setTimeout(this.checkVisibility, 50)
+    }
+  }
+
+  componentWillUnmount () {
+    clearTimeout(this.timer)
+  }
+
+  checkVisibility = () => {
+    const { bannerStaticRef } = this.props
+
+    if (bannerStaticRef) {
+      const bannerTop = bannerStaticRef.getBoundingClientRect().top + 100
```

Comment:
What is this magic `100`?

Reply:
situation: when insight has all viewport height and static banner is showing only for 5-10px. I think user should see enough part of static banner. 
Or, if you mean that it should be as variable, I can't come up with a name for this number))

Reply:
Sure, make sense 👍.
Proposed variable name: `BOTTOM_BANNER_EXTRA_VIEWPORT`